docs/index.md

```diff
@@ -78,6 +78,18 @@ It's possible to configure the behaviour of certain resources using the followin
 
 * `default_naming_suffix` - (Optional) The default name suffix to create the azure resource. Used together with `name` in each resource block. Conflicts with `default_name`. Changing this forces new resources to be created.
 
+* `endpoint` - (Optional) A `endpoint` block as defined below.
```

Comment:
The naming here is bit of confused, shall we change them to align with the structure of the Go SDK?

Reply:
It's aligned with azure cli: https://learn.microsoft.com/en-us/cli/azure/manage-clouds-azure-cli#list-available-clouds
and raw response from the metadata url.

Reply:
I've tried this https://management.azure.com/metadata/endpoints?api-version=2022-09-01, which has different structure than the CLI one, which I felt confused..